third_party/move/move-model/src/ast.rs

```diff
@@ -451,20 +451,29 @@ pub enum ExpData {
     /// some other source into expressions. Consumers of expressions should assume this
     /// variant is not present and can panic when seeing it.
     Invalid(NodeId),
+
     /// Represents a value.
     Value(NodeId, Value),
+
     /// Represents a reference to a local variable introduced in the AST.
     LocalVar(NodeId, Symbol),
+
     /// Represents a reference to a temporary used in bytecode, if this expression is associated
     /// with bytecode.
+    /// Seems to also be used to represent a parameter to a function, where TempIndex indicates the
```

Comment:
Can we not add 'Seems' comments? Also I find this comment rather unnecessary. A parameter is a temporary as well. Temporaries are called so because they have temporary lifetime on the stack.

Reply:
In bytecode a parameter is a temporary.  In AST from compiled Move code there is no stack.

The existing comment is inconsistent with the observed usage.  Therefore I've added a comment to warn other readers about other usages.

However, I don't know that I've understood completely, so I've included "seems" to indicate that there is some uncertainty about this, especially whether it's complete.